view/css/emulator.css

```diff
@@ -272,11 +282,28 @@ hr {
     user-select: none;
 }
 
+#codeedit {
```

Comment:
Same as above, this can be part of your future MR for this new feature :smile:

Reply:
#removed